draft-ietf-rtcweb-jsep.xml

```diff
@@ -631,10 +631,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         target="RFC5888"></xref>, Section 4, to identify
                         the m= line. JSEP implementations creating an
                         ICE Candidate object MUST populate both of these
-                        fields. Implementations receiving an ICE
+                        fields, using the MID of the associated RtpTransceiver
+                        object (which may be locally generated when interacting
+                        with a remote non-JSEP endpoint
+                        that does not support the MID attribute).
+                        Implementations receiving an ICE
```

Comment:
This seems almost right. Maybe ("which may be locally generated by the answerer") and then link to the section below where you explain that.